### PR TITLE
[Backport 1.11] fix(PodStorageConfigSection): fix variable reference

### PR DESCRIPTION
**Pull request by Poltergeist**
This commit fixes a variable reference which lead to have multiple lines in the volume mount table
with the same values.
Before:
image

After:
image

To test:

```JSON
{
  "id": "/host-volume",
  "volumes": [
    {
      "name": "etc",
      "host": "/etc"
    }
  ],
  "scaling": {
    "kind": "fixed",
    "instances": 1
  },
  "scheduling": {
    "upgrade": {
      "minimumHealthCapacity": 0,
      "maximumOverCapacity": 0
    }
  },
  "containers": [
    {
      "name": "container1",
      "exec": {
        "command": {
          "shell": "sleep 5000"
        }
      },
      "resources": {
        "cpus": 0.1,
        "mem": 128,
        "disk": 0,
        "gpus": 0
      },
      "image": {
        "id": "alpine",
        "kind": "DOCKER",
        "forcePull": false
      },
      "volumeMounts": [
        {
          "name": "etc",
          "mountPath": "/mnt/etc1",
          "readOnly": true
        }
      ],
      "lifecycle": {
        "killGracePeriodSeconds": 60
      }
    },
    {
      "name": "container2",
      "exec": {
        "command": {
          "shell": "sleep 5000"
        }
      },
      "resources": {
        "cpus": 0.1,
        "mem": 128,
        "disk": 0,
        "gpus": 0
      },
      "image": {
        "id": "alpine",
        "kind": "DOCKER",
        "forcePull": false
      },
      "volumeMounts": [
        {
          "name": "etc",
          "mountPath": "/mnt/etc2",
          "readOnly": true
        }
      ],
      "lifecycle": {
        "killGracePeriodSeconds": 60
      }
    }
  ]
}
```
Go to the create multiple container service form and open the JSON Editor and add the previously added JSON.
Then hit review and run if everything is okay the table at the bottom should show two different volume mounts.

Closes DCOS-19219

### CODE REVIEW

File: tests/pages/services/ServicePodReviewScreen-cy.js [hunk 1]
```diff
@@ -1063,6 +1063,259 @@ describe("Services", function() {
...
+      cy.contains("Multi-container (Pod)").click();
+
+      cy
+        .root()
+        .getFormGroupInputFor("Service ID *")
+        .type(`{selectall}{rightarrow}${serviceName}`);
+
+      cy.get(".menu-tabbed-item").contains("container-1").click();
+
+      // TODO: Due to a bug in cypress you cannot type values with dots
```
Comment (nLight) [inline tests/pages/services/ServicePodReviewScreen-cy.js]: We've upgraded cypress, shall we try uncommenting this part? ;)

File: tests/pages/services/ServicePodReviewScreen-cy.js [hunk 2]
```diff
@@ -1063,6 +1063,259 @@ describe("Services", function() {
       });
     });
 
+    it("renders proper review screen and JSON for a pod with two containers and ephemeral volume", function() {
+      const serviceName = "pod-with-ephemeral-volume";
+      const command = "`while true ; do echo 'test' ; sleep 100 ; done";
+      const cmdline = "while true; do echo 'test' ; sleep 100 ; done";
```
Comment (nLight) [inline tests/pages/services/ServicePodReviewScreen-cy.js]: These two lines look very similar to me 🤔

File: tests/pages/services/ServicePodReviewScreen-cy.js (hunk 1 above)
Comment (Poltergeist) [reply tests/pages/services/ServicePodReviewScreen-cy.js]: Then we should do this for all the tests.

Comment (DanielMSchmidt) [reply tests/pages/services/ServicePodReviewScreen-cy.js]: Let's add a follow up PR for that 👍

Comment (nLight) [reply tests/pages/services/ServicePodReviewScreen-cy.js]: C'mon! It's a new test!

Comment (nLight) [reply tests/pages/services/ServicePodReviewScreen-cy.js]: Let's please fix the new one and fix all the others in the followup

Comment (Poltergeist) [reply tests/pages/services/ServicePodReviewScreen-cy.js]: New ©️ 🍝

**Comment by nLight** [comment]
Thank you for addressing the minor requests 🙇

**Pull request merged**